Models/Event.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace WsuTriangle.AttendanceTracker.Models;
+
+public class Event
```

Comment:
Events have Attendance. As a navigational property, you should add:
```
public ICollection<Attendance> Attendances { get; set; }
```

Then, you may query:
```
var eventAttendance = db.Events.Include(x => x.Attendances).Where(x => x.EventId == SOME_ID).Attendances.ToList()
```